code/game/area/desertdam.dm

```diff
@@ -188,12 +192,9 @@
 /area/desert_dam/interior/caves/northern_caves
 	name = "Northern Caves"
 	icon_state = "red"
-	minimap_color = MINIMAP_AREA_CAVES
 /area/desert_dam/interior/caves/east_caves
 	name = "Eastern Caves"
 	icon_state = "red"
-	minimap_color = MINIMAP_AREA_CAVES
-
```

Comment:
why do you hate spacing haha

Reply:
Added the gorillion missing spaces. God damn chug has so many areas.